lib/src/main/java/graphql/nadel/engine/transform/partition/NadelPartitionKeyExtractor.kt

```diff
@@ -17,19 +17,4 @@ interface NadelPartitionKeyExtractor {
         inputValueDef: GraphQLInputValueDefinition,
         context: NadelFieldPartitionContext,
     ): String?
-
-    companion object {
-        /**
-         * A no-op partition key extractor that does not partition any scalar values.
-         */
-        val noop: NadelPartitionKeyExtractor = object : NadelPartitionKeyExtractor {
```

Comment:
No reason for a default noop. If the consumer code wants to implement the hook, it should really provide a real implementation, otherwise there's no point in implementing the hook at all.

Reply:
I assume the remaining `internal` impl is just for testing?

Edit: ah I see it's used in the default no-op impl